app/assets/styles/css/page-headers.css

```
@@ -4,10 +4,8 @@
   padding: 0;
   color: var(--white-color);
   position: relative;
-  background-position: center center;
   background-size: cover;
-  background: var(--body-text-alpha-20);
-  background: linear-gradient(0deg, var(--body-text-alpha-20), var(--primary-color-alpha-60));
+  background: linear-gradient(0deg, var(--body-text-alpha-20), var(--primary-color-alpha-60)) center center;
```

Biome (2.1.2)

[error] Unexpected shorthand property background after background-size



(lint/suspicious/noShorthandPropertyOverrides)